simapp/v2/simdv2/cmd/commands.go

```diff
@@ -278,10 +278,4 @@ func registerGRPCGatewayRoutes[T transaction.Tx](
 	cmtservice.RegisterGRPCGatewayRoutes(deps.ClientContext, server.GRPCGatewayRouter)
 	_ = nodeservice.RegisterServiceHandlerClient(context.Background(), server.GRPCGatewayRouter, nodeservice.NewServiceClient(deps.ClientContext))
 	_ = txtypes.RegisterServiceHandlerClient(context.Background(), server.GRPCGatewayRouter, txtypes.NewServiceClient(deps.ClientContext))
-
-	for _, mod := range deps.ModuleManager.Modules() {
-		if gmod, ok := mod.(module.HasGRPCGateway); ok {
-			gmod.RegisterGRPCGatewayRoutes(deps.ClientContext, server.GRPCGatewayRouter)
```

Comment:
A nice follow-up of this is to start de-wire grpc gateway from module on main, by removing

```diff
version: v1
plugins:
  - name: gocosmos
    out: ..
    opt: plugins=grpc,Mgoogle/protobuf/any.proto=github.com/cosmos/gogoproto/types/any
-  - name: grpc-gateway
-    out: ..
-    opt: logtostderr=true,allow_colon_final_segments=true
``` 

And remove all generated `*.gw.go` from modules and `RegisterGRPCGatewayRoutes`.
It will need to wait 1. of https://github.com/cosmos/cosmos-sdk/issues/22904 but that's very soon.